src/Lean/Util/Path.lean

```diff
@@ -30,13 +30,19 @@ abbrev SearchPath := System.SearchPath
 namespace SearchPath
 
 /-- If the package of `mod` can be found in `sp`, return the path with extension
-`ext` (`lean` or `olean`) corresponding to `mod`. Otherwise, return `none.` -/
+`ext` (`lean` or `olean`) corresponding to `mod`. Otherwise, return `none`. Does
+not check whether the returned path exists. -/
 def findWithExt (sp : SearchPath) (ext : String) (mod : Name) : IO (Option FilePath) := do
```

Comment:
We should probably rename it to something like `getPathOfModuleFile?` or something... but not in this PR